test/e2e/network_segmentation.go

```diff
@@ -2117,7 +2117,7 @@ spec:
     topology: Layer3
     layer3:
       role: Primary
-      subnets: ` + generateCIDRforClusterUDN()
+      subnets: ` + generateCIDRforClusterUDN("10.20.100.0/16", "2014:100:200::0/60")
```

Comment:
ive seen 10.x have conflicts downstream with the CDN from other tests using 10.x - any chance you can make it a rarer IP range?

Reply:
buuuut downstream :D we changed it to 103.103 we don't use this in origin.. again these are netseg tests which exist but I can change them no harm.. for bgp I am using the rare 103.103

Reply:
i am thinking of test extentions but i guess i can update my PR which is doing that and change this 10.20.